dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/DotTemplateTool.java

```diff
@@ -37,32 +38,17 @@
  * Created by Jonathan Gamba
  * Date: 10/25/12
  */
-public class DotTemplateTool implements ViewTool {
+public class DotTemplateTool  {
 
-    private static HttpServletRequest request;
-    Context ctx;
-    private static User sysUser = null;
+    //private static HttpServletRequest request;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)